docs/DeveloperGuide.md

```diff
@@ -659,6 +659,43 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 <hr>
 
+**System**: FitBook
+
+**Use case**: UC05 - Edit a client's details
+
+**Actor**: User
+
+**MSS**
+
+1. User requests to view list of all clients.
+1. FitBook shows a list of clients.
+1. User requests to edit the details of a specific client in the list.
+1. FitBook edits the details of a client in the list.
+1. FitBook displays a success message after the details of the client is successfully edited.
+1. Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+    * Use case ends.
+
+* 3a. The given index is invalid.
+    * 3a1. FitBook shows an error message.
+    * Step 3a1 is repeated until the user gives a valid index.
+    * Use case resumes at step 4.
+
+* 3b. User enters an invalid command.
```

Comment:
Invalid command format should be used here. Invalid command would imply an unknown command instead.